include/gz/launch/config.hh.in

```diff
@@ -1,19 +1,41 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
 /* Config.hh. Generated by CMake for @PROJECT_NAME@. */
 
+#ifndef GZ_LAUNCH_CONFIG_HH_
+#define GZ_LAUNCH_CONFIG_HH_
+
 /* Version number */
-#define IGNITION_LAUNCH_MAJOR_VERSION ${PROJECT_VERSION_MAJOR}
-#define IGNITION_LAUNCH_MINOR_VERSION ${PROJECT_VERSION_MINOR}
-#define IGNITION_LAUNCH_PATCH_VERSION ${PROJECT_VERSION_PATCH}
+#define GZ_LAUNCH_MAJOR_VERSION ${PROJECT_VERSION_MAJOR}
+#define GZ_LAUNCH_MINOR_VERSION ${PROJECT_VERSION_MINOR}
+#define GZ_LAUNCH_PATCH_VERSION ${PROJECT_VERSION_PATCH}
 
-#define IGNITION_LAUNCH_VERSION "${PROJECT_VERSION}"
-#define IGNITION_LAUNCH_VERSION_FULL "${PROJECT_VERSION_FULL}"
+#define GZ_LAUNCH_VERSION "${PROJECT_VERSION}"
+#define GZ_LAUNCH_VERSION_FULL "${PROJECT_VERSION_FULL}"
 
-#define IGNITION_LAUNCH_VERSION_HEADER "${PROJECT_NAME_LOWER}, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2019 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
+#define GZ_LAUNCH_VERSION_HEADER "Gz Launch, version ${PROJECT_VERSION_FULL}\nCopyright (C) 2019 Open Source Robotics Foundation.\nReleased under the Apache 2.0 License.\n\n"
 
-#define IGNITION_LAUNCH_INITIAL_CONFIG_PATH "${CMAKE_INSTALL_PREFIX}/${IGN_DATA_INSTALL_DIR}/configs"
+#define GZ_LAUNCH_INITIAL_CONFIG_PATH "${CMAKE_INSTALL_PREFIX}/${IGN_DATA_INSTALL_DIR}/configs"
 
-#define IGNITION_LAUNCH_PLUGIN_INSTALL_PATH "${IGN_LAUNCH_PLUGIN_INSTALL_PATH}"
+#define GZ_LAUNCH_PLUGIN_INSTALL_PATH "${IGN_LAUNCH_PLUGIN_INSTALL_PATH}"
 
 #cmakedefine BUILD_TYPE_PROFILE 1
 #cmakedefine BUILD_TYPE_DEBUG 1
 #cmakedefine BUILD_TYPE_RELEASE 1
```

Comment:
Did you mean to remove these like you did for other libs?

Reply:
![image](https://user-images.githubusercontent.com/20265670/170592805-1485b855-b628-497a-9cf9-2c92d2dd2289.png)
They're removed on mine o__O

git isn't reflecting it???

Reply:
I'm going to "flush" git with a dummy commit. It might dirty the history a bit but I think it's worth it

Reply:
https://github.com/gazebosim/gz-launch/pull/163/commits/584c5b3f431793ec9a72fd1cd8dc23b069362dfd
https://github.com/gazebosim/gz-launch/pull/163/commits/4e0953480a371a6b8af85dd2ffea49acd284e40f

Flushed! I've never seen something like this before

EDIT: I think the changes aren't reflecting because the "Files Changed" page is on an old set of commits o__O

![image](https://user-images.githubusercontent.com/20265670/170593389-b08e332a-4b50-46f2-8464-e89933481f53.png)

Reply:
Ahhh pardon!